Libraries/CustomComponents/ListView/ListView.js

```diff
@@ -184,6 +185,18 @@ var ListView = React.createClass({
      */
     renderSectionHeader: PropTypes.func,
     /**
+     * (sectionID) => style object
+     *
+     * If provided, set custom style on the section
+     * It can be a function returning a style or an object
+     * Default : null
+     */
+    sectionStyle: React.PropTypes.oneOfType([
+      PropTypes.func,
```

Comment:
Not sure we want this -- "style" means a style object everywhere else in React Native